CMakeLists.txt

```diff
@@ -228,6 +263,7 @@ endif()
 # add all native executables
 add_subdirectory(src/bin)
 add_subdirectory(src/gmmbin)
+add_subdirectory(src/fgmmbin)

```

Comment:
We normally exclude fgmmbin because it's basically never used.  In regular Makefile it's in an additional target "ext".
Might be a little wasteful to build it...